### ISSUE TITLE
Automatic Import from Other Launchers

### ISSUE THREAD

**Issue by kreezxil** [gorilla-devs/GDLauncher#524]
**Is your feature request related to a problem?**
I see users complaining about that when they manually copy their instances from Twitch or another launcher, that GDL can't see their instances. I understand why as does the team. 

**Describe the solution you'd like**
As you're aware when you install new web browsers, they ask to import bookmarks and other data from other browsers on the system. Doing that in GDL for other launchers would be a godsend. 

**Describe alternatives you've considered**
Telling users to export the pack in Twitch and the importing the custom export. The problem is that when they have say more than 5 such custom packs they don't want to do that even tho that would solve the problem.

**Additional context**
Having GDL doing something like this automatically would be a GODSEND, even if it were an external tool that could be offered to do that. Referring them to said tool would be very helpful.

Basically a mass custom export with mass custom import.

**Comment by dajix350** [comment]
How do I up vote this epic suggestion?

**Comment by kreezxil** [comment]
spread the word to all gdl users and ask them to comment and react to it.
